index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+    
+  -->
+    <!-- <h1>Hello World, we are software developers!</h1> -->
+
+    <main id="main-doc">
+      Technical documentation
+      <section class="main-section" id="topic_1">
+        <header>Topic 1</header>
+        <!-- <a href="#topic_1" class="nav-link">Topic_1</a> -->
+        <p></p>
+        <p></p>
+        <code></code>
+        <li></li>
+      </section>
+      <section class="main-section" id="topic_2">
+        <header>Topic 2</header>
+        <!-- <a href="#topic_2" class="nav-link">Topic_2</a> -->
+        <p></p>
+        <p></p>
+        <code></code>
+        <li></li>
+      </section>
+      <section class="main-section" id="topic_3">
+        <header>Topic 3</header>
+        <!-- <a href="#topic_3" class="nav-link">Topic_3</a> -->
```

Comment:
Please make sure that the unused anchor tags are removed from the html codes.

Reply:
> Please make sure that the unused anchor tags are removed from the html codes.

- [ ] Kindly provide a reason as to why you are requiring this change